workflows/variantmtb.nf

```diff
@@ -41,16 +39,13 @@ workflow VARIANTMTB {
     */
 
     // CHECK PARAMETERS
-    if ( params.databases.contains("cgi"    )    & !params.cgi_cancer_type          )   { error("Please include the cancer types to query CGI for!" )}
```

Comment:
Do you check for this now via the config?

Reply:
Or is that obsolete anyways because it defaults to 'Any cancer type'?

Reply:
the whole parameter `cgi_cancer_type` is obsolete/got moved to the samplesheet